packages/sdk/wallets/wallet-base/src/signing-utils.test.ts

```diff
@@ -602,7 +602,7 @@ describe('recoverTransaction', () => {
     expect(recoverTransaction(signed)[1]).toEqual(account.address)
   })
 
-  test.only('cip42 serialized by viem (v=1)', async () => {
+  test('cip42 serialized by viem (v=1)', async () => {
```

Comment:
we were skipping a bunch of tests 😱

Reply:
good catch! could you also change it [here](https://github.com/celo-org/developer-tooling/blob/07c4c7854f419dd07fbf09fe966fb5b378a139d1/packages/cli/src/commands/validatorgroup/deregister.test.ts#L113) please?